src/Services/DatabaseTools/AbstractDatabaseTool.php

```diff
@@ -221,4 +230,22 @@ protected function getCacheMetadataParameter()
         return $this->container->hasParameter(self::CACHE_METADATA_PARAMETER_NAME)
             && false !== $this->container->getParameter(self::CACHE_METADATA_PARAMETER_NAME);
     }
+
+    /**
+     * Compatibility layer for Symfony <= 4.3
```

Comment:
do we even want to still care about <- 4.3? https://symfony.com/releases

Reply:
or would not including this create a BC break?

Reply:
well, currently, the composer.json says it supports Symfony 3.4+.

Reply:
I would prefer to support Symfony 3.4 as long as it doesn't cause too many trouble.

Reply:
ok, fine for me!

Reply:
Finally, I changed my mind and removed support of Symfony < 4.4. Symfony 3.4 will only get security fixes now, I think no one would upgrade that bundle while keeping a legacy version of Symfony.